docs/DeveloperGuide.md

```diff
@@ -154,90 +155,44 @@ Classes used by multiple components are in the `seedu.address.commons` package.
 
 This section describes some noteworthy details on how certain features are implemented.
 
-### \[Proposed\] Undo/redo feature
+Usage of `editClient` and `editMeeting`:
 
-#### Proposed Implementation
+`editClient` and `editMeeting` execute in a similar manner to each other.
+Let the term entity refer to either a client or a meeting.
+Every entity is uniquely identified by a UUID in storage, so essentially all fields in an entity can be edited without loss of uniqueness of the entity.
 
-The proposed undo/redo mechanism is facilitated by `VersionedMyInsuRec`. It extends `MyInsuRec` with an undo/redo history, stored internally as an `myInsuRecStateList` and `currentStatePointer`. Additionally, it implements the following operations:
+[Proposed] Multiple possible prefixes per command feature
 
-* `VersionedMyInsuRec#commit()` — Saves the current MyInsuRec state in its history.
-* `VersionedMyInsuRec#undo()` — Restores the previous MyInsuRec state from its history.
-* `VersionedMyInsuRec#redo()` — Restores a previously undone MyInsuRec state from its history.
+In this proposed feature, the user is provided with  multiple possible prefixes for defining fields in a command.
+For example, currently we can define a client's birthday using the `\b` prefix.
+However, since a birthday is essentially a date, a user may prefer to reuse the `\d` prefix instead (see `addMeeting` command).
```

Comment:
Is it just me or is the prefix used here are in the wrong order?
Example, `\b` and `\d` shouldn't it be `b/` and `d/`?

Reply:
oh noes, thank you for mentioning this.